src/templates/formio.html

```diff
@@ -27,5 +27,8 @@
       builder="builder"
       options="options"
     ></formio-component>
+    <div ng-repeat="alert in formioAlerts track by $index" class="alert alert-{{ alert.type }}" role="alert" ng-if="::!builder">
```

Comment:
These alerts should probably be above the submit button. It may look weird to have them below the submit button.

Reply:
okay, Travis, I will put them above the submit button and make another pull request.

Reply:
@travist Are you sure we want it above the submit button? That might make it odd since the submit button is a component and may/may not be on a form and may/may not be at the bottom.

Reply:
Also, are we really sure we want to do this? Wouldn't it be better to have a configuration to say where the alerts show up? We should have an option of "top", "bottom" or "both". I'd bet some people don't want the errors to show up in both.

Reply:
One final note, this can be overridden inside an app anyway.